x-pack/plugins/security_solution/public/common/containers/alerts/use_alert_prevalence_from_process_tree.ts

```diff
@@ -79,15 +91,20 @@ function useAlertDocumentAnalyzerSchema(processEntityId: string, indices: string
   }
 }
 
-export function useAlertPrevalenceFromProcessTree(
-  processEntityId: string,
-  timelineId: string | undefined
-): UserAlertPrevalenceFromProcessTreeResult {
+export function useAlertPrevalenceFromProcessTree({
+  processEntityId,
+  documentId,
+  timelineId,
+  indices,
+}: UseAlertPrevalenceFromProcessTree): UserAlertPrevalenceFromProcessTreeResult {
   const http = useHttp();
 
   const { selectedPatterns, to, from } = useTimelineDataFilters(timelineId);
-
-  const { loading, id, schema } = useAlertDocumentAnalyzerSchema(processEntityId, selectedPatterns);
+  const alertAndOriginalIndices = [...new Set([...selectedPatterns, ...indices])];
```

Comment:
```suggestion
  const alertAndOriginalIndices = [...new Set([selectedPatterns.concat(indices)])];
```
💅  Saves one spread operation